packages/radix-ui-themes/src/components/segmented-control.css

```diff
@@ -20,12 +20,11 @@
 
   /* Create a new stacking context */
   isolation: isolate;
+}
 
-  &.disabled {
-    cursor: var(--cursor-disabled);
-    color: var(--gray-a8);
-    background-color: var(--gray-3);
-  }
+.rt-SegmentedControlRoot:where([data-disabled]) {
```

Comment:
Not idea why, but when nesting the selector, one of our postcss plugins is moving this declaration before the parent declaration. Un-nesting fixes this, so we don't need extra specificity to override this.

Also: using `data-disabled` handles `cursor` for us since that's set everywhere by our base styles.